modules/apps/27-interchain-accounts/controller/ibc_middleware.go

```diff
@@ -163,3 +167,22 @@ func (im IBCModule) OnTimeoutPacket(
 
 	return im.app.OnTimeoutPacket(ctx, packet, relayer)
 }
+
+// SendPacket implements the ICS4 Wrapper interface
+func (im IBCMiddleware) SendPacket(
```

Comment:
Ok I understand now why you've done it this way.

I think it should be documented in this function's go-doc that the packet is going to get constructed directly by ICA controller and sent to IBC rather than being created by ICA-auth.

Reading this file, made me think SendPacket wasn't happening at all when really it is happening through the controller's SendTx function